Cargo.toml

```diff
@@ -69,7 +70,9 @@ sqlitedb = ["dep:rusqlite"]
 # these are always required, so easier to specify this way
 default = [ "aes", "hmac", "pbkdf2", "sqlitedb" ]
 
-standard = [ "ecdsa", "ec_montgomery", "eddsa", "ecdh", "hash", "hkdf", "rsa", "sp800_108", "sshkdf", "tlskdf"]
+ecc_all = ["ecdsa", "ec_montgomery", "eddsa", "ecdh"]
+
+standard = [ "ecc_all", "hash", "hkdf", "rsa", "sp800_108", "sshkdf", "tlskdf"]
 
 fips = [ "rusqlite/bundled", "ecdsa", "ecdh", "hash", "hkdf", "rsa", "sp800_108", "sshkdf", "tlskdf"]
```

Comment:
I think we wanted to enable eddsa for FIPS too, didn't we?

Reply:
Yes eventually, but to do that we need either a new version of OpenSSL, or we need to add selftests ourselves, so "not yet".